### DISCUSSION TITLE
Why next and previous buttons not working

### DISCUSSION THREAD

**Discussion by surushTodzhibekov** [igordanchenko/yet-another-react-lightbox#187] [General]
I have more than one photo but When I click on next or previous icon it is not navigating to the next photo. I appreciate if someone help me!
     `                    <>
                    <img
                      className={styles.img}
                      src={file.data}
                      alt={file.name}
                      onClick={() => setOpen(true)}
                    />
                    <Lightbox
                      open={open}
                      close={() => setOpen(false)}
                      controller={{ closeOnBackdropClick: true }}
                      index={index}
                      slides={files.map((file) => ({ src: file.data }))}
                    />
                  </>`

**Comment by igordanchenko** [comment]
Please consider providing a minimal reproducible example as the above snippet isn't really sufficient to troubleshoot your issue.
